pkg/etcdutil/layout.go

```diff
@@ -64,12 +63,12 @@ func TaskMasterPath(appName string, taskID uint64) string {
 	return path.Join("/", appName, TasksDir, strconv.FormatUint(taskID, 10), TaskMaster)
 }
 
-func MetaPath(linkType, appName string, taskID uint64) string {
+func MetaPath(appName string, taskID uint64) string {
 	return path.Join("/",
 		appName,
 		TasksDir,
 		strconv.FormatUint(taskID, 10),
-		linkType)
+		"meta")
```

Comment:
I agree with hongchao.
When introduce master-worker mechanism, I think we should separate topology,meta mechanism into two part. 
I also think the construction of topology is so redundant. I prefer a interface like
GetLink(srcID uint64, srcType string, dstType string, relationship string) []uint64
to construct this topology. User only implement topology by describing the relationship. 
The return type []uint64 points the serial number in dstNumber, and our internal topology implementation could follow this relationship automatically. 
As a result, the user need not to implement different topology over epochs. They only should do is to specify the node num of every task in every epoch.